tests/integrations/tests/test_google.py

```
@@ -751,6 +751,84 @@ def test_11_integration_specific_features(self, google_client, test_config):
 
         assert_valid_chat_response(response3)
 
+    
+    @pytest.mark.parametrize("provider,model", get_cross_provider_params_for_scenario("simple_chat"))
+    def test_11a_system_instruction(self, google_client, test_config, provider, model):
```

Ruff (0.14.10)

Unused method argument: `test_config`

(ARG002)